### ISSUE TITLE
Reddit's minimize comment line

### ISSUE THREAD

**Issue by v1rTu0Zz** [philc/vimium#3038]
So new reddit design makes you click on a tinny thread line left to the comment you want to minimize and vimium doesn't offer me a shortcut for that, it offers few that are on it but when I press them nothing happens, seems like they only select the comment right to them and not minimize it. I know that this is a really small issue so I'm not very optimistic about a solution but I trust a few others could appreciate it as well. Thanks in advance/

**Comment by smblott-github** [comment]
I don't know the particular example, but it could be that that element has an onclick listener, so we're not picking it up.

We have code to detect onlick listeners and make them clickable, but it was removed (actually, disabled) because it broken some web (important) we sites.

We don't currently have a replacement solution.

**Comment by alphaCTzo7G** [comment]
Yeah.. the new reddit makes it very hard to navigate with `vimium`. Since `reddit` is 3rd most popular website, (probably more among programmers, who are probably heavy users of chrome), maybe `vimium` should optimize a little bit to improve the user experience on `reddit`?

**Comment by pmhgms** [comment]
I found a quick solution
Install tampermonkey or something similar
And add these code for reddit
```
setTimeout(function(){
    var e = document.getElementsByClassName("threadline");
    for(var i = 0; i < e.length; i++){
        var d = document.createElement('button');
        d.setAttribute("class","threadline");
        e[i].parentNode.replaceChild(d, e[i]);
    }
}, 3500);
```

or just install it from [here](https://greasyfork.org/en/scripts/387100-reddit-threadline-for-vimium)
